libraries/lib-datahandler/src/main/thrift/projects.thrift

```diff
@@ -67,6 +73,7 @@ struct Project {
     13: optional ProjectState state,
     15: optional ProjectType projectType,
     16: optional string tag,// user defined tags
+    17: optional ProjectClearingState clearingState,
```

Comment:
This works:

![tmp](https://cloud.githubusercontent.com/assets/1187050/21174146/b691a06e-c1db-11e6-9fbe-bd6b2622d3ad.png)